objtesting/foreach.go

```diff
@@ -4,6 +4,7 @@
 package objtesting
 
 import (
+	"github.com/thanos-io/objstore/providers/obs"
```

Comment:
nit: Can we keep imports grouping? i.e. move this down to the last group.

Reply:
sure can,I will fix it.